### PR TITLE
added doc strings and linted

### PR DESCRIPTION
**Pull request by jamesonthecrow**
Probably some merge conflicts now, but here is a branch with doc strings. I also linted things to the PEP8 standard as much as I could.

### CODE REVIEW

File: sentenai/api.py [hunk 1]
```diff
@@ -380,80 +541,109 @@ def dataframe(self, only=None):
         else:
             return output
 
+    def _mIdx(self, df):  # NOQA
```
Comment (stites) [inline sentenai/api.py]: NOQA? Is this something needed for flake8?

**Comment by stites** [comment]
@jamesonthecrow 👍 very nice! Can you resolve the conflict?

File: sentenai/__init__.py [hunk 2]
```diff
@@ -23,13 +23,14 @@ def event(*args, **kwargs):
 
 
 def ast(q):
+    """Print the query as an Abstract Synatx Tree."""
```
Comment (kyleseaman) [inline sentenai/__init__.py]: typo - `Syntax`

**Comment by jamesonthecrow** [comment]
This should be all set to merge now!

File: sentenai/flare.py [hunk 3]
```diff
@@ -319,80 +498,179 @@ def __str__(self):
...
+
+        Used primarily to escape segments of paths that would be invalid
+        in the host language. For example, if a path segment contains `:
+
+        >>> evt.foo._('...').bar.bat
+
+        Arguments:
+            name -- the name of the variable to get.
+        """
+<<<<<<< HEAD
```
Comment (stites) [inline sentenai/flare.py]: Looks like the merge conflict wasn't resolved properly.

Comment (jamesonthecrow) [reply sentenai/flare.py]: Oops. Forgot to save changes to this file. Should be fixed now.

**Comment by stites** [comment]
LGTM, :tada: for documentation!

**Pull request merged**